frontend/src/components/filter/Fuels.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
if you can import it, then you installed it : your PR lacks the modification of package.json and package-lock.json